org-gtd-engage.el

```diff
@@ -80,14 +80,18 @@
         (org-agenda nil "g"))))
 
 ;;;###autoload
-(defun org-gtd-show-all-next ()
+(defun org-gtd-engage-next ()
   "Show all next actions from all agenda files in a single list.
 This assumes all GTD files are also agenda files."
   (interactive)
   (org-gtd-core-prepare-agenda-buffers)
   (with-org-gtd-context
       (org-todo-list org-gtd-next)))
 
+;;;###autoload
+(defalias 'org-gtd-show-all-next 'org-gtd-engage-next)
+(make-obsolete 'org-gtd-show-all-next "use `org-gtd-engage-next' instead." "3.0.0")
```

Comment:
Maybe, instead of giving the version here as 3.0.0 I should instead use a date string?